tools/debug-ui/README.md

```diff
@@ -39,3 +39,9 @@ same keys are only fetched once.
 It's also helpful to understand at a high level how the react-router library works; this is used
 to support deep-linking in the URL (e.g. `/127.0.0.1/cluster` leads to the cluster page), allowing
 the UI to be served as a single application.
+
+### Linting & Formatting
+The project is configured to use ESLint (error-checking) and Prettier (consistent formatting).
+
+Run `npm run lint` to check for linting & formatting errors, and `npm run fix` to fix those that
+can be automatically fixed.
```

Comment:
Great idea to describe it in a README!

Optional suggestion: Consider also adding a JS/TS section in `docs/practices/style.md` so it becomes visible in the general style guide that contributors will look at here: https://near.github.io/nearcore/practices/style.html

Reply:
Thanks! Let me do that in a separate PR (also need to think about what exactly to write there).